Projects/COM/COM04/COM04-AWS200.md

```diff
@@ -0,0 +1,53 @@
+# COM04-AWS200 - Create a cluster of virtual machines using docker swarm
+
+## Cloud Service Provider
+
+- Amazon Web Services
+
+## Difficulty
+
+- Level 200 (Intermediate)
+
+## Project's Author(s)
+
+[Karan Gauswami](https://github.com/KaranGauswami)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create a VPC for our EC2 Instances
+- Create an EC2 Instance with initial script to install Docker
+- Setup that EC2 Instance as a manager node
+- Create multiple instances (4 to 5) with initial script to create worker nodes and join that manager node
```

Comment:
- 2-3 instances should be enough. If above two, the number of instances does not influence the amount of learning achieved. The big number required can scare away people $$$ (this was my first thought too when I read it).
- "Initial script" is called "User Data" on AWS, very important detail - If that's what it refers to.

Everything else looks good👍

Reply:
Changed Instance count from 4-5 to 2-3